### DISCUSSION TITLE
New Ticket on FromCreator

### DISCUSSION THREAD

**Discussion by DooSys** [glpi-project/glpi-agentmonitor#10] [Ideas]
Hi

Firstly Glpi Agent Monitor is a great improvement for the agent.
For FormCreator users it would be possible to modify the destination of the "New Ticket..." function.

![image](https://github.com/glpi-project/glpi-agentmonitor/assets/32511461/d9271fb2-4e65-4bf2-83d8-09a493c171af)

Aujourd'hui http://*****/glpi/front/ticket.form.php en http://******/glpi/marketplace/formcreator/front/formlist.php

Thanks ;)

**Comment by redddcyclone** [comment]
Hi @DooBio!

There is already an [open issue](https://github.com/redddcyclone/glpi-agentmonitor/issues/13) about this in the development repository.

I will be working on it in the near future.

Thanks!
